Mathlib/LinearAlgebra/TensorProduct/Vanishing.lean

```diff
@@ -209,16 +239,22 @@ theorem vanishesTrivially_iff_sum_tmul_eq_zero_of_rTensor_injective
 Assume that every expression $\sum_i m_i \otimes n_i$ which vanishes also vanishes trivially.
 Then, for every submodule $M' \subseteq M$, the map $M' \otimes N \to M \otimes N$ is injective. -/
 theorem rTensor_injective_of_forall_vanishesTrivially
-    (hMN : ∀ {ι : Type u} [Fintype ι] {m : ι → M} {n : ι → N},
+    (hMN : ∀ {ι : Type uι} [Fintype ι] {m : ι → M} {n : ι → N},
       ∑ i, m i ⊗ₜ n i = (0 : M ⊗[R] N) → VanishesTrivially R m n)
```

Comment:
For lemmas like this that assumes a condition of the form `∀ {ι : Type uι} [Fintype ι]`, we also should replace `ι` by `Fin n`.